CHANGELOG.md

```diff
@@ -1,5 +1,36 @@
 # Changelog
 
+## [1.8.0] - 2023-09-13
+
+### Added
+
+- `azure_key_vault` KeyManager plugin (#4458)
+- Server configuration to set refresh hint of local bundle (#4400)
+- Support for batch entry deletion in `spire-server` CLI (#4371)
+- `ca_journals` database table (#4371)
```

Comment:
I'd leave this out. It won't have any user visible impact until we implement the rest of the journal migration.

Reply:
Hmm, I feel like we should call out any database changes since we allow users to disable automatic database migrations.

Reply:
Also just noticed, this should be #4465, not #4371.

Reply:
We have included database schema changes in the CHANGELOG in the past, but I think that we don't usually do that, particularly when they are in preparation for a future feature (which are the most cases) and there is nothing exposed to the user yet.
It's common that we make a lot of changes that are not user-facing until the feature is complete and we turn it on (one example is the force rotation work, where we are introducing a lot of changes across different releases). Rather than including those changes in each release, we include the feature in the CHANGELOG when it's introduced and we point to all the PRs involved.

I would probably leave this out and include this PR in the PRs list when we mention the feature in the release. Having this in the CHANGELOG before may lead to confusions.

Reply:
That seems reasonable, I just want to make sure the requirement to perform a database migration gets captured appropriately. If it makes more sense to do that when we make the feature user-facing, I'm good with that.

Maybe in the future we should add a `Database Changes` section to our release notes to make this more apparent for users? I guess users would probably need to read the code to know when they need to perform a manual migration.